packages/utilities/src/utilities/markdownconverter/markdownconverter.js

```diff
@@ -0,0 +1,53 @@
+const _htmlTagRegex = /<.*?>/g;
+const _doubleSpaceRegex = /\s{2,}/g;
+const _italicRegex = /[_*](.*?)[_*]/g;
+const _boldRegex = /[_*]{2}(.*?)[_*]{2}/g;
+
+/**
+ *
+ * @param {string} str String to be checked for html tags
+ * @returns {string} String with html tags stripped out
+ */
+const _removeHtmlTags = str => str.replace(_htmlTagRegex, '');
+
+/**
+ *
+ * @param {string} str String to be checked for double spaces
+ * @returns {string} String with html double spaces fixed
+ */
+const _fixDoubleSpaces = str => str.replace(_doubleSpaceRegex, ' ');
+
+/**
+ * Utiltity function for converting markdown into html
+ *
+ * @param {string} str String to convert to html
+ * @param {object} options Object with options for the conversion
+ * @param {boolean} options.italic Defines if should convert italic
+ * @param {boolean} options.bold Defines if should convert bold
```

Comment:
Are these default as true or false?

Reply:
These are default as false, but then if none is declared it means all "styles" are applied. May be a little controversial in the way i coded it out but it seemed the simpler way to do it. I could default it to true and and change the logic though... Do you think that would be better?

Reply:
I honestly don't have a preference either way. Perhaps true by default? Otherwise passing nothing doesn't do anything, right?

Reply:
Exactly, that's why I chose this approach, it was the one that most sense from the adopter experience pov. Any other approach would take an extra level of complexity that I'm not would be good for the adopter, be it adding extra flag to allow allow, accepting string/array of strings and so on...